modules/node-tuning-operator.adoc

```diff
@@ -19,12 +23,24 @@ endif::operators[]
 ifdef::perf[]
 = About the Node Tuning Operator
 endif::perf[]
+ifdef::cluster-caps[= Node Tuning capability]
+
 ifdef::operators[]
 [discrete]
 == Purpose
 endif::operators[]
+
```

Comment:
All the other sections in this doc still have this Purpose title like [here](https://57602--docspreview.netlify.app/openshift-enterprise/latest/installing/cluster-capabilities.html#marketplace-operator_cluster-capabilities). I think that title should be added for consistency. It should just be something like
```
 [discrete]
== Purpose
```
on line 32. It might work adding your parameter `cluster-caps` to the ifdef thats there, but we'd have to check that the title size is consistent. You could also look at the other modules that do this for reference, ex: https://raw.githubusercontent.com/openshift/openshift-docs/main/modules/operator-marketplace.adoc